fc-community-extension/src/contentModifiers.js

```diff
@@ -163,11 +163,14 @@ const makeFactchainHtmlNote = (note) => {
 }
 
 const addNote = async (mainArticle, note) => {
-  let htmlNote = makeFactchainHtmlNote(note);
+  
+  const provider = await createFactchainProvider();
+  const userAddress = await provider.getAddress();
+  const author = userAddress.toLowerCase() === note.creatorAddress.toLowerCase() ? "You" : "Factchain users"
+  let htmlNote = makeFactchainHtmlNote(note, author);
+  
   if (!note.finalRating) {
-    const provider = await createFactchainProvider();
-    const userAddress = await provider.getAddress();
-    if (!userAddress || userAddress.toLowerCase() !== note.creatorAddress.toLowerCase()) {
+    if (!userAddress || author !== "You") {
```

Comment:
A few things to fix in the whole `addNote` function:

- `userAddress` is not necessarily set, it can be undefined, in which case we would fail to even show the note
- using `author !== "You"` is not great, maybe have a `isAuthor` that can be used in different locations?
- Defining `"You" : "Factchain users"` here is not great, would be better inside of `makeFactchainHtmlNote`. Which means you could probably move `<div>${rateItHtml}</div>` to `makeFactchainHtmlNote` as well, keeping all the html in one place
- Below we add a listener on `#rateNoteButton`, but it does not necessarily exist, which creates an exception.